### PR TITLE
fix: channel custom fields

### PR DESCRIPTION
**Pull request by peterdeme**
For Strava, Lombok weirdness

### CODE REVIEW

**Review by ferhatelmas — APPROVED** [review]
LGTM

File: .github/CODEOWNERS [hunk 1]
```diff
@@ -1 +1 @@
-* @ferhatelmas @miagilepner @driver-devel
+* @ferhatelmas @driver-devel
```
Comment (ferhatelmas) [inline .github/CODEOWNERS:1]: yourself please

Comment (peterdeme) [reply .github/CODEOWNERS]: ✅

**Review by ferhatelmas — APPROVED** [review]

**Comment by ferhatelmas** [comment]
good to go for a release

**Pull request merged**